website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Getting Started With The dbt Semantic Layer"
```

Comment:
```suggestion
title: "Getting started with The dbt Semantic Layer"
```

Optional if you wanted to change the title capitalization to be more consistent with dbt Labs standard!

Reply:
Updated and also changed The to the